gene2phenotype_project/gene2phenotype_app/serializers/user.py

```diff
@@ -11,6 +14,7 @@ class UserSerializer(serializers.ModelSerializer):
     email = serializers.CharField(read_only=True)
     panels = serializers.SerializerMethodField()
     is_active = serializers.CharField(read_only=True)
+    username = serializers.CharField(write_only=True)
```

Comment:
We already have the username. 
L13 `user_name = serializers.SerializerMethodField()`